router/a.com.conf

```diff
@@ -0,0 +1,15 @@
+server {
+    listen 443 ssl;
+    server_name a.com;
+    server_tokens off;
+
+    ssl_certificate /data/nginx/certs/origin_a_com.crt.pem;
+    ssl_certificate_key /data/nginx/certs/origin_a_com.key.pem;
+
+    location / {
+        proxy_pass  http://localhost:8888;
```

Comment:
What does `8888` port stands for? Our app is serving traffic on `8080` port. Do we have anything additional?

Reply:
The spring application before changes started on port 80, but now on port 80 I gave nginx so I changed the port of the application itself to 8888 and the traffic goes from nginx to port 8888